flattening-json-messages/README.adoc

```diff
@@ -1,4 +1,4 @@
-= Flattening JSON Messages with KSQL
```

Comment:
I think the suffix is useful here, because it's the kind of thing people will google for. 
Maybe
`Flattening JSON Kafka Messages` instead?